Gems/Atom/Tools/AtomToolsFramework/Code/Source/Util/Util.cpp

```diff
@@ -761,15 +759,27 @@ namespace AtomToolsFramework
             });
     }
 
-    AZStd::vector<AZStd::string> GetNonCacheSourceFolders()
+    bool IsPathIgnored(const AZStd::string& path)
+    {
+        for (const auto& pattern : GetSettingsObject("/O3DE/Atom/Tools/IgnoredPathPatterns", AZStd::vector<AZStd::string>{}))
+        {
+            if (AZ::StringFunc::Contains(path, pattern))
```

Comment:
This could be misleading. `AZ::StringFunc::Contains` is not a pattern matching function.